server/db.js

```diff
@@ -10,5 +10,14 @@ const connectDB = async () => {
 
   console.log(`MongoDB Connected: ${conn.connection.host}`.cyan.underline.bold);
 };
+const Schema  = mongoose.Schema;
+const ProfileSchema =  new Schema({ 
+		FirstName: String,
+		LastName: String,
+		Description: String,
+		Availability: 	Date
 
-module.exports = connectDB;
+	 });
+const Profile = mongoose.model('Profile', ProfileSchema);
+
+module.exports = { connectDB, Profile }; 
```

Comment:
Might be a good idea to extract models in their own file/folder, otherwise looks good!

Reply:
Resolved.